docs/api.md

```diff
@@ -842,6 +842,8 @@ The normal users that can connect to radon with password.
 
 ### create user
 
+The api used to create user for radon, not for backends or backup. 
+
```

Comment:
Here we should make it clear that the API user is used to obtain authentication credentials for connecting to Radon Server, it's a localhost user only select privileges

Reply:
changed